components/post_view/commented_on/index.ts

```diff
@@ -29,9 +43,9 @@ function mapStateToProps(state, ownProps) {
     };
 }
 
-function mapDispatchToProps(dispatch) {
+function mapDispatchToProps(dispatch: Dispatch<GenericAction>) {
     return {
-        actions: bindActionCreators({
+        actions: bindActionCreators<ActionCreatorsMapObject<ActionFunc>, Actions>({
             showSearchResults,
             updateSearchTerms,
```

Comment:
Hi @devinbinnie, I need help to type this one, `updateSearchTerms` does not have the signature of a typical action so this line results in a type error. I'm not sure how to type it.

Btw, I don't see any of the actions being used or propagated to the children. Is it safe to just remove it?

Reply:
Huh, weird. Seems like we would have removed those if they weren't used, but you're right there's no reference to them as far as I can see. Feel free to remove them :)

Is there any other assistance you need now that those can be removed?

Reply:
No thanks, shouldn't be a problem.